### ISSUE TITLE
Agent Status: Unable to Focus `Toggle Chat` using keyboard navigation

### ISSUE THREAD

**Issue by benibenj** [microsoft/vscode#289340]
<img src="https://github.com/user-attachments/assets/e0624402-332f-42b5-a1ea-cb58b67f3ac4" alt="Image" width="185" height="50">
